### PR TITLE
Google Analytics institutional pageviews (12090)

### PR DESCRIPTION
**Pull request by sallain**
This adds documentation for an improvement to Google Analytics
tracking for AtoM sites. Users can now configure Analytics to
track which institution a page belongs to, so that the Analytics
administrator can see an institution's pageviews at a glance. The
documentation covers the following:

* how to configure Google Analytics to record institution name data
* how to view institution name data in the Google Analytics Dashboard pageview report
* how to export pageview data with institution names from the Google Analytics Dashboard to a CSV file

### CODE REVIEW

**Comment by mcantelon** [comment]
Not sure if this needs to be addressed in docs, but I think Radda's work on institutional pageviews also has the positive side effect of updating our analytics from a depricated version (see https://projects.artefactual.com/issues/6090).

File: admin-manual/maintenance/web-analytics.rst [hunk 1]
```diff
@@ -30,22 +30,96 @@ These types of tools require access to the log files generated by the web server
...
+
+In the file config/app.yml, set the newly-created dimension index number below
+the API key and make sure that the setting is uncommented.
+
+.. code-block:: yaml
+
+     google_analytics_api_key: UA-XXXXX-X
+     google_analytics_institutions_dimension_index: 1
+
+Remember to clear the cache and restart PHP-FPM after configuration.
```
Comment (jraddaoui) [inline admin-manual/maintenance/web-analytics.rst]: Not sure if we have specific section(s) in the doc to link to explain how to do this, but it'd be nice if we have them.

File: admin-manual/maintenance/web-analytics.rst [hunk 2]
```diff
@@ -30,22 +30,96 @@ These types of tools require access to the log files generated by the web server
...
 
 Replace UA-XXXXX-X with your tracking ID. Once you are done, remember to
 :ref:`clear the cache <maintenance-clear-cache>`.
+
+Google Analytics for tracking institutional pageviews
++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+As of AtoM 2.5, it is possible to track pageviews for individual institutions
+who are sharing a multi-repository AtoM instance by adding a custom dimension to
+the tracking ID. Adding this dimension will set the authorized form of name from
```
Comment (jraddaoui) [inline admin-manual/maintenance/web-analytics.rst:59]: I think it will be nice to say it's the **source culture** auth. form of name.

**Review by jraddaoui — APPROVED** [review]
Looks great @sallain! And good catch @mcantelon!

Just a couple of suggestions.

**Comment by sallain** [comment]
@mcantelon good catch - the docs don't mention how Analytics is implemented, so I'm going to leave out the specifics, but we can definitely close that public ticket!

@radda thanks, added both those suggestions

**Pull request merged**